designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputPhoneNumber.kt

```diff
@@ -41,14 +41,15 @@ fun InputPhoneNumber(
     isRequiredField: Boolean = false,
     onNextClicked: (() -> Unit)? = null,
     onValueChanged: ((String?) -> Unit)? = null,
+    onFocusChanged: ((Boolean) -> Unit) = {},
```

Comment:
why don't null as the BasicTextfield?

Reply:
No reason if I'm honest, I just added it there rather than in all the examples, I'll rectify it